src/components/StatusStrip/StatusStrip.test.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { mount } from "enzyme";
+
+import StatusStrip from "./StatusStrip";
+
+describe("Status Strip", () => {
+  it("should show correct status counts", () => {
+    const fakeStatusList = {
+      model: [
+        { label: "foo", count: 1 },
+        { label: "bar", count: 2 },
+        { label: "baz", count: 3 },
+      ],
+    };
+    const wrapper = mount(<StatusStrip statusList={fakeStatusList} />);
+    const StatusStripHeading = wrapper.find(".status-strip strong");
+    expect(StatusStripHeading.text()).toBe("6 models:");
+    expect(wrapper.find(".is-foo").text()).toBe("foo: 1");
+    expect(wrapper.find(".is-bar").text()).toBe("bar: 2");
+    expect(wrapper.find(".is-baz").text()).toBe("baz: 3");
+  });
+
+  it("should show label correctly pluralized", () => {
```

Comment:
This doesn't actually test that anything gets pluralized. You'll need to add another test case for 2 models.

Reply:
The test case above with 6 models tests that the label should be '6 models'. This test has one model and tests that the label is '1 model'.

Reply:
It's better to have clear tests that test a specific aspect of the code. I would be surprised if the test "should show correct status counts" failed and "should show label correctly pluralized" tested due to pluralize issue. 

To resolve, can you remove the `expect(StatusStripHeading.text()).toBe("6 models:");` assertion in the test above and add a second dataset to this test please. It would be more helpful for future us.